### PR TITLE
(maint) Add test and release actions

### PR DESCRIPTION
**Pull request by underscorgan**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: scarhand/actions-ruby@master
+      - name: Build gem
+        args: build *.gemspec
+      - name: Publish gem
+        env:
+          RUBYGEMS_AUTH_TOKEN: ${{ secrets.RUBYGEMS_AUTH_TOKEN }}
```
Comment (mwaggett) [inline .github/workflows/release.yml]: Where does this live?

Comment (underscorgan) [reply .github/workflows/release.yml]: https://github.com/puppetlabs/packaging/settings/secrets

Comment (mwaggett) [reply .github/workflows/release.yml]: cool! Didn't know about that!

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        args: push *.gem
```
Comment (e-gris) [inline .github/workflows/release.yml]: globs. why does it have to be globs? ;-)

**Pull request merged**